### PR TITLE
Release 0.26.1

### PR DESCRIPTION
**Pull request by emilk**
Some bug fixes!

### CODE REVIEW

**Comment by emilk** [comment]
@rustbasic can you please test if this branch still fixes your image-interaction bugs?

I had to roll back part of my fix to not break semver.

File: examples/hello_world/src/main.rs [hunk 1]
```diff
@@ -49,9 +49,20 @@ impl eframe::App for MyApp {
             }
             ui.label(format!("Hello '{}', age {}", self.name, self.age));
 
-            ui.image(egui::include_image!(
-                "../../../crates/egui/assets/ferris.png"
-            ));
+            // ui.image(egui::include_image!(
```
Comment (YgorSouza) [inline examples/hello_world/src/main.rs:52]: Is this change to the example intentional? It looks like test code.

Comment (emilk) [reply examples/hello_world/src/main.rs:52]: oops! thanks for catching that 😅

**Comment by rustbasic** [comment]
Yes.
Once the v0.26.1 PR is finished, should that test it?

**Comment by emilk** [comment]
I think this is ready to go (I need to generate changelogs etc first ofc), but I would appreciate you testing it @rustbasic since you've been very good at finding bugs

**Comment by rustbasic** [comment]
This is good.
I tested it with the contents of the current master branches, and there is no problem.
  You did a really good job.

**Comment by rustbasic** [comment]
@emilk 

Bug : 
When you open the Viewport, the speed drops to 1 frame per second.

**Comment by rustbasic** [comment]
@emilk 

In egui v0.26.1, many bugs have been improved, and the overall feeling of use is good.
You did a really good job.

**Pull request closed without merging**